lib/lago/__init__.py

```diff
@@ -620,8 +643,8 @@ def _ova_to_spec(self, filename):
 
         if image_file is not None:
             disk_meta = {"root-partition": "/dev/sda1"}
-
-            disk_spec = [{"type": "file",
+            disk_spec = [{"type": "template",
+                          "template_type": "file",
```

Comment:
That's not true, before you had a qcow2 file, that you used directly, and modified it in place, now you create a new one each time with that old one as backing file so you end up with two+ qcows, and not modifying the first one that is extracted from the ova file.
Afaik the file is not removed at the end of the run, and it might be reused (it will only redownload if it's not there), so it's not temporary right?